### PR TITLE
upgrade to 4.1.2

### PR DESCRIPTION
**Pull request by Thovi98**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Thovi98** [comment]
!testme

**Comment by Thovi98** [comment]
@ericgaspar

File: scripts/install [hunk 1]
```diff
@@ -15,12 +15,18 @@ source /usr/share/yunohost/helpers
 
 random_key=$(ynh_string_random --length=32)
 email=$(ynh_user_get_info --username=$admin --key="mail")
+fpm_footprint="low"
+fpm_free_footprint=0
+fpm_usage="low"
```
Comment (ericgaspar) [inline scripts/install:20]: I wonder if we should worry about these settings for Monica.
Also, if you think we should keep it, the config panel is disabled. We should remove the `.example` at that end.

Comment (Thovi98) [reply scripts/install:20]: Oh no, I don't think we need it!
I only tried to make monica installable again, we can indeed remove this

**Comment by Thovi98** [comment]
!testme

File: scripts/install (hunk 1 above)
Comment (Thovi98) [reply scripts/install]: Okay I’ve tried but I don’t know what to remove haha, I broke everything :p

**Comment by ericgaspar** [comment]
merging, thanks for the work!

**Pull request merged**